test/spawn.jl

```diff
@@ -469,3 +479,19 @@ let c = `ls -l "foo bar"`
     @test length(c) == 3
     @test eachindex(c) == 1:3
 end
+
+## Deadlock in spawning a cmd (#22832)
+# FIXME?
```

Comment:
Mentioned in the merge commit text. I don't know the answers to any of those questions, so I can't provide more information. If I knew any of those answers, I would have already fixed the test.